app/base/models.py

```diff
@@ -22,9 +22,22 @@ class User(db.Model, UserMixin):
     )
     user_prop = db.relationship("Property", backref="prop_owner", lazy=True)
 
+    def to_json(self):
+        """
+        This function returns the a json representation of the object. This is useful if we want to include and
```

Comment:
a typo here? `...the a...`

and  maybe have the wording on each line and a dot at end
```
This function returns the a json representation of the object. 
This is useful if we want to include and return data related to the instance from other database tables.
```

Reply:
It was just a typo